snuba/pipeline/single_query_plan_pipeline.py

```diff
@@ -0,0 +1,46 @@
+from snuba.pipeline.query_pipeline import (
+    QueryPipeline,
+    QueryPipelineBuilder,
+    _execute_query_plan_processors,
+)
+from snuba.datasets.plans.query_plan import (
+    ClickhouseQueryPlan,
+    ClickhouseQueryPlanBuilder,
+    QueryRunner,
+)
+from snuba.request import Request
+from snuba.web import QueryResult
+
+
+class SingleQueryPlanPipeline(QueryPipeline):
+    """
+    A query pipeline for a single query plan.
```

Comment:
Please add a note with a TODO on the fact that we are going to eventually incorporate the whole query processing sequence in a pipeline.